synapse/storage/databases/main/pusher.py

```diff
@@ -388,6 +393,57 @@ def _delete_pushers(txn) -> int:
 
         return number_deleted
 
+    async def _remove_deleted_email_pushers(
+        self, progress: dict, batch_size: int
+    ) -> int:
+        """A background update that deletes all pushers for email addresses no longer
+        associated with a user.
```

Comment:
Could you note here in [Google python docstring notation](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) a description for each argument and the return value? Something analogous to: https://github.com/matrix-org/synapse/blob/684d19a11c3b93c9dd5fb90f43d38aa7e8c6005f/synapse/storage/database.py#L1795-L1805